Document map-item-frame-cursor-update-interval (#100)
PaperMC/PaperDocs: source/server/configuration.rst

* **description**: The rate, in ticks, at which the server updates containers
  and inventories.

map-item-frame-cursor-update-interval
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
* **default**: 10

> * **description**: The interval in ticks for which cursors on maps in item frames are updated.
>   A number less than 1 would disable updates all together.

Reviewer: ```suggestion
* **description**: The interval in ticks at which cursors on maps in item frames are updated.
  Setting this to a number less than 1 will disable updates all together.
```

I think it would sound better like this, but I might be wrong.

>   Setting this to a number less than 1 will disable updates all together.

Reviewer: ```suggestion
  Setting this to a number less than 1 will disable updates altogether.
```

I missed one more thing, `all together` = *all at once*, or *all at the same time*-- `altogether` = *completely* like you want here.
